eric_gonzalez/Polls.csv

```diff
@@ -0,0 +1,61 @@
+The Hill/HarrisXThe Hill,1/20 - 1/22,878 RV,29,17,9,11,5,2,4,4,2,--,1,Biden +12
+FOX NewsFOX News,1/19 - 1/22,495 LV,26,23,14,10,7,3,5,3,0,0,1,Biden +3
+Economist/YouGovEconomist,1/19 - 1/21,470 LV,28,18,21,6,8,4,3,2,3,0,0,Biden +7
+MonmouthMonmouth,1/16 - 1/20,372 RV,30,23,14,9,6,5,3,1,1,1,0,Biden +7
+CNNCNN,1/16 - 1/19,500 RV,24,27,14,5,11,4,4,2,0,0,0,Sanders +3
+Morning ConsultM. Consult,1/15 - 1/19,12402 LV,29,24,15,10,8,3,4,3,2,1,0,Biden +5
+SurveyUSASUSA,1/14 - 1/16,1086 LV,32,21,14,9,9,2,4,3,2,--,--,Biden +11
+The Hill/HarrisXThe Hill,1/13 - 1/14,451 RV,29,19,11,7,4,3,2,3,1,2,0,Biden +10
+Economist/YouGovEconomist,1/11 - 1/14,521 RV,27,20,19,5,7,3,3,1,2,0,0,Biden +7
+QuinnipiacQuinnipiac,1/8 - 1/12,651 RV,25,19,16,6,8,4,5,1,1,1,1,Biden +6
+Politico/Morning ConsultPolitico,1/6 - 1/12,17096 RV,29,23,14,8,8,3,5,4,2,1,0,Biden +6
+IBD/TIPPIBD/TIPP,1/3 - 1/11,333 RV,26,15,20,7,9,3,3,2,1,0,0,Biden +6
+Economist/YouGovEconomist,1/5 - 1/7,574 RV,27,20,22,3,7,3,3,2,3,0,0,Biden +5
+Morning ConsultM. Consult,12/30 - 1/5,17213 RV,31,23,14,7,8,3,4,4,2,1,0,Biden +8
+Harvard-HarrisHarris,12/27 - 12/29,756 RV,30,17,12,7,7,2,3,2,1,--,--,Biden +13
+Economist/YouGovEconomist,12/28 - 12/31,548 RV,29,19,18,3,8,4,3,2,3,0,0,Biden +10
+The Hill/HarrisXThe Hill,12/27 - 12/28,431 RV,28,16,11,11,6,2,2,2,2,0,0,Biden +12
+Politico/Morning ConsultPolitico,12/23 - 12/29,17787 RV,32,21,14,6,8,3,4,3,1,1,0,Biden +11
+Economist/YouGovEconomist,12/22 - 12/24,586 LV,30,17,19,4,7,5,3,1,2,0,0,Biden +11
+Politico/Morning ConsultPolitico,12/20 - 12/22,7178 RV,31,21,15,6,9,3,5,3,2,1,0,Biden +10
+EmersonEmerson,12/15 - 12/17,525 RV,32,25,12,3,8,2,6,2,4,0,0,Biden +7
+NBC News/Wall St. JrnlNBC/WSJ,12/14 - 12/17,410 LV,28,21,18,4,9,5,3,1,2,0,0,Biden +7
+Economist/YouGovEconomist,12/14 - 12/17,555 LV,29,19,17,4,7,4,3,2,3,1,0,Biden +10
+CNNCNN,12/12 - 12/15,408 RV,26,20,16,5,8,3,3,1,1,0,0,Biden +6
+The Hill/HarrisXThe Hill,12/13 - 12/14,456 RV,29,13,13,5,5,3,3,3,1,1,2,Biden +16
+QuinnipiacQuinnipiac,12/11 - 12/15,567 RV,30,16,17,7,9,3,3,1,1,0,0,Biden +13
+USA Today/SuffolkUSA Today,12/10 - 12/14,384 LV,23,14,13,6,8,3,2,1,1,0,--,Biden +9
+Politico/Morning ConsultPolitico,12/9 - 12/15,13384 RV,31,22,15,7,8,2,4,3,2,1,0,Biden +9
+NPR/PBS/MaristNPR/PBS,12/9 - 12/11,704 RV,24,22,17,4 ,13,4 ,5 ,0,1 ,1 ,0,Biden +2
+FOX NewsFOX News,12/8 - 12/11,453 RV,30,20,13,5,7,5,3,1,3,1,0,Biden +10
+IBD/TIPPIBD/TIPP,12/5 - 12/14,312 RV,26,18,14,5,9,2,2,2,1,1,0,Biden +8
+Economist/YouGovEconomist,12/7 - 12/10,497 LV,26,16,21,4,11,2,3,1,3,1,0,Biden +5
+QuinnipiacQuinnipiac,12/4 - 12/9,665 RV,29,17,15,5,9,3,4,1,2,1,0,Biden +12
+MonmouthMonmouth,12/4 - 12/8,384 RV,26,21,17,5,8,4,3,1,0,0,1,Biden +5
+Politico/Morning ConsultPolitico,12/2 - 12/8,15442 RV,30,22,16,6,9,2,4,3,2,1,0,Biden +8
+Economist/YouGovEconomist,12/1 - 12/3,541 LV,27,13,18,3,12,3,2,0,2,1,1,Biden +9
+The Hill/HarrisXThe Hill,11/30 - 12/1,437 RV,31,15,10,6,9,2,2,2,0,1,1,Biden +16
```

Comment:
I don't think this file should be in the repo

Reply:
Should be fine since someone will probably need this if they want to build the project locally, unless this shouldn't be "public"

Reply:
I believe Eric had this in here as an example CSV.  Probably doesn't need to be in here unless, as Chris said, someone wants to build locally.